swagger.json

```diff
@@ -3760,6 +3804,44 @@
                 }
             }
         },
+        "DataInputHistoryResponse": {
+            "type": "object",
+            "required": [
+                "name"
+            ],
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description":"The ID of this data input",
+                    "example": 12345
+                },
+                "name": {
+                    "type": "string",
+                    "description":"Name of this data input",
+                    "example":"Pump Flow"
+                },
+                "points": {
+                    "type": "array",
+                    "description": "Data points from this data input",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "timeMs":{
+                                "type":"integer",
+                                "format": "int64",
+                                "example": 1453449599999
+                            },
+                            "value":{
+                                "type":"integer",
```

Comment:
Should type be "number" instead of "integer"? The integer + float combination looks a little suspect.

Reply:
done!